crates/junction-api/src/kube/http.rs

```diff
@@ -55,11 +66,14 @@ impl Route {
     /// Convert a Gateway API [HTTPRoute][gateway_http::HTTPRoute] into a Junction [Route].
     pub fn from_gateway_httproute(httproute: &gateway_http::HTTPRoute) -> Result<Route, Error> {
         let id = Name::from_str(httproute.meta().name.as_ref().unwrap()).unwrap();
-        let (hostnames, ports) =
-            from_parent_refs(httproute.spec.parent_refs.as_deref().unwrap_or_default())?;
+        let namespace = httproute.meta().namespace.as_deref().unwrap_or("default");
```

Comment:
I'd be ok returning an error here as well. It seems like we should never be parsing an HTTPRoute that has no namespace on it.